components/indexPage/SideGoodsCard.vue

```diff
@@ -60,6 +60,11 @@ import Rubbishbin from "@/assets/svg/rubbish-bin.svg?inline";
 export default {
   components: { Penciledit, Rubbishbin },
   props: ["sideitem"],
+  computed: {
+    isRootName() {
+      return this.$route.name === 'index' ? 'hidden' : '';
+    }
+  },
```

Comment:
@jacquesme there is no need of computed prop here, it will be computed on on component loading, so it could be moved to data:
```
data() {
   return {
       isRootName: this.$route.name === 'index' ? 'hidden' : ''
   }
}
```

Reply:
Not sure about naming: `isRootName` looks like boolean and not string class.

Reply:
@jacquesme and if you want to completely hide something then better to use `v-if` then toggle class to avoid element rendering.